proxy-node-acceptance-tests/features/smoke/integration.feature

```diff
@@ -1,7 +0,0 @@
-Feature: eidas-proxy-node-smoke-test-integration feature
```

Comment:
We're sure it's safe to remove this?

Reply:
It runs off the release/single-country-legacy branch, so we are assuming so